thoth-errors/src/database_errors.rs

```diff
@@ -120,6 +120,9 @@ static DATABASE_CONSTRAINT_ERRORS: Map<&'static str, &'static str> = phf_map! {
     "series_series_url_check" => "Invalid series URL.",
     "subject_subject_code_check" => "Subject codes must not be an empty string.",
     "subject_subject_ordinal_check" => "A subject ordinal number must be greater than 0.",
+    "title_unique_canonical_true_idx" => "Caninical title of current work already exists.",
+    "title_title_check" => "Title must not be an empty string.",
```

Comment:
you should also add the other checks in the table (title_subtitle_check, and title_full_title_check). Also you can remove the existing errors for these three fields under "work_" (work_title_check, work_full_title_check, work_subtitle_check)